releasetasks/__init__.py

```diff
@@ -33,13 +36,17 @@ def make_task_graph(root_template="release_graph.yml.tmpl", template_dir=DEFAULT
         "never": arrow.now().replace(years=1000),
         "get_complete_mar_url": lambda a, b, c, d: "COMPLETE MAR URL",
         "get_complete_mar_artifact": lambda a, b, c: "COMPLETE MAR ARTIFACT",
-        # TODO: this should be a hash of the revisions in the push
-        "revision_hash": "abcdef",
-        "get_treeherder_platform": lambda p: p,
+        # Treeherder expects 12 symbols in revision
+        "revision_hash": th.get_resultsets(
+            template_kwargs["branch"],
+            revision=template_kwargs["revision"][:12])[0]["revision_hash"],
+        "get_treeherder_platform": treeherder_platform,
         # TODO: unstub these
-        "encrypt_env_var": lambda a, b, c, d, e: "ENCRYPTED",
+        "encrypt_env_var": encryptEnvVar_wrapper,
```

Comment:
Do the changes to these end up with any changes to the generated graphs? If so, you should probably add some tests around that.